xlgui/collection.py

```diff
@@ -186,7 +181,7 @@ def on_force_rescan_button_clicked(self, widget):
         from xlgui import main
         main.mainwindow().controller.on_rescan_collection_forced()
         
-    
+    @GtkTemplate.Callback
```

Comment:
Oh, sorry. I should just start exaile with `--debug`, otherwise I won't see these issues.